examples/legacy_basic_reconnect.py

```diff
@@ -0,0 +1,58 @@
+import sys
+import time
+from niveristand.errors import RunError
+from niveristand.legacy import NIVeriStand
+
+# This is an example script to demonstrate the usage of NIVeriStand Python API.
+# This script requires Python 32b(tested on Python 3.9.6) and NIVeriStand Python API.
```

Comment:
I think it should work with python 64 bit as well, you can remove this requirement